### PR TITLE
Feature/game class

### PR DESCRIPTION
**Pull request by noahdurbin**
## Type Of Change
- [X] New Feature
- [ ] Debugging
- [ ] Refactor/File Clean-up
- [X] Tests
- [ ] Documentation

## Provide A Summary Of Your Change
In this Branch I added functionality to the game class so that we can create a game object along with tests for this. I then worked in the game factory to have a method that creates game objects from csv data.

## Group Project Check List
- [X] I made test for my code, and made sure they were passing before submitting PR
- [X] I assigned atleast 2 group members to review my PR

### CODE REVIEW

**Comment by noahdurbin** [comment]
If there is a way that we can only accept some of the changes and not include the coverage files that would be great, didn't mean to include that, and couldn't figure out how to remove it from the commit.

File: lib/game_factory.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class GameFactory
+  attr_reader :games
+
+  def initialize
+    @games = []
+  end
+
+  def create_games(path)
+    CSV.foreach((path), headers: true, header_converters: :symbol) do |row|
+      @games << Game.new(
```
Comment (zach-bergman) [inline lib/game_factory.rb:10]: I like how you did this, I setup all of the info for the GameTeam object in a hash, then passed that hash in to the GameTeam.new(). I can refactor if this is how we'd like to have it across all factories.

File: lib/game_factory.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+class GameFactory
+  attr_reader :games
+
+  def initialize
+    @games = []
```
Comment (austincarrjones) [inline lib/game_factory.rb:5]: I like the way you're creating a collection here. I'll mimic this, it's more dynamic than how I was copying the class lesson (magical_pets) which worked but wasn't accumulating the objects in this fashion

File: spec/game_spec.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      expect(game1.season).to be_a(Integer)
+      expect(game1.type).to be_a(Symbol)
+      expect(game1.date_time).to be_a(Integer)
+      expect(game1.away_team_id).to be_a(Integer)
+      expect(game1.home_team_id).to be_a(Integer)
+      expect(game1.away_goals).to be_a(Integer)
+      expect(game1.home_goals).to be_a(Integer)
+      expect(game1.venue).to be_a(String)
+      expect(game1.venue_link).to be_a(String)
+    end
```
Comment (zach-bergman) [inline spec/game_spec.rb:24]: Nice testing!

**Review by austincarrjones — APPROVED** [review]
This will be helpful for me to refactor my team_factory file.

**Review by zach-bergman — APPROVED** [review]
I approve! Will push up my branch soon.

**Pull request merged**